app/editor/src/features/admin/tags/TagListFilter.tsx

```diff
@@ -16,7 +16,7 @@ export const TagListFilter: React.FC<IAdminFilterProps> = ({ setGlobalFilter })
         <IconButton
           iconType="plus"
           label={`Add new tag`}
-          onClick={() => navigate(`/admin/tags/0`)}
+          onClick={() => navigate(`/admin/tags/NEW`)}
```

Comment:
Let's hope they never want to create a NEW tag lol

Reply:
Lol was thinking about that...